sgkit/curate.py

```diff
@@ -0,0 +1,75 @@
+from typing import Hashable
+
+import xarray as xr
+from xarray import Dataset
+
+from sgkit import variables
+from sgkit.utils import conditional_merge_datasets
+
+
+def filter_partial_calls(
+    ds: Dataset,
+    *,
+    call_genotype: Hashable = variables.call_genotype,
+    merge: bool = True,
+) -> Dataset:
+    """Replace partial genotype calls with missing values.
+
+    Parameters
+    ----------
+    ds
+        Genotype call dataset such as from
+        :func:`sgkit.create_genotype_call_dataset`.
+    call_genotype
+        Input variable name holding call_genotype as defined by
+        :data:`sgkit.variables.call_genotype_spec`
+    merge
+        If True (the default), merge the input dataset and the computed
+        output variables into a single dataset, otherwise return only
+        the computed output variables.
+        See :ref:`dataset_merge` for more details.
+
+    Returns
+    -------
+    Dataset containing updated variables `call_genotype` and
+    `call_genotype_mask` in which partial genotype calls are replaced
+    with compleately missing genotype calls.
+
+    Examples
+    --------
+
+    >>> import warnings
+    >>> import sgkit as sg
+    >>> from sgkit.testing import simulate_genotype_call_dataset
```

Comment:
`simulate_genotype_call_dataset` is in the top-level namespace now, so you can just reference as `sg. simulate_genotype_call_dataset`